src/interface/src/styleguide/scenario-card/scenario-card.component.ts

```diff
@@ -43,42 +46,88 @@ export type ScenarioStatus = 'Running' | 'Done' | 'Failed';
   styleUrl: './scenario-card.component.scss',
 })
 export class ScenarioCardComponent {
-  @Input() status: ScenarioStatus = 'Running';
+  @Input() status: ScenarioResultStatus | undefined = 'SUCCESS';
   @Input() name = '';
-  @Input() areas = 0;
-  @Input() budget = 0;
-  @Input() treatmentPlansCount = 0;
-  @Input() creator = '';
-  @Input() created_at = '';
+  @Input() areas? = 0;
+  @Input() budget? = 0;
+  @Input() treatmentPlansCount?: number | undefined = 0;
+  @Input() creator?: string | undefined = '';
+  @Input() created_at? = '';
+  @Input() selected?: boolean = false;
 
   @Output() openScenario = new EventEmitter();
   @Output() openPlanningProgress = new EventEmitter();
   @Output() archiveScenario = new EventEmitter();
+  @Output() clicked = new EventEmitter();
 
-  readonly chipsStatus: Record<ScenarioStatus, StatusChipStatus> = {
-    Done: 'success',
-    Running: 'running',
-    Failed: 'failed',
+  readonly chipsStatus: Record<ScenarioResultStatus, StatusChipStatus> = {
+    FAILURE: 'failed',
+    LOADING: 'running',
+    NOT_STARTED: 'running',
+    PANIC: 'failed',
+    PENDING: 'running',
+    RUNNING: 'running',
+    SUCCESS: 'success',
+    TIMED_OUT: 'failed',
+  };
+
+  readonly chipLabel: Record<ScenarioResultStatus, ScenarioResultLabel> = {
```

Comment:
I'm tempted to write this as `Record<StatusChipStatus, ScenarioResultLabel>`, because this is just a difference of capitalization and "Done" instead of "success", but not sure what's easiest to maintain.

Reply:
Maybe something like
```
export type ScenarioStatus = 'FAILURE' | 'LOADING' | 'SUCCESS' | 'PENDING';
// ...
// (...)
// ...
  readonly chipsStatus: Record<
    ScenarioStatus,
    { status: StatusChipStatus; label: string }
  > = {
    FAILURE: { status: 'failed', label: 'Failed' },
    PENDING: { status: 'inProgress', label:'In progress'}
    SUCCESS: { status: 'success', label:'Great success'}
  };
```

Though ScenarioStatus should probably live somewhere else, on scenario.types?